src/openforms/logging/models.py

```diff
@@ -93,3 +125,19 @@ def message(self) -> str:
         return self.get_message()
 
     message.short_description = _("message")
+
+
+class TimelineLogProxyManager(models.Manager):
+    def get_queryset(self):
+        qs = super().get_queryset()
+        return qs.filter(extra_data__contains={"avg": True})
+
+
+class AvgTimelineLogProxy(TimelineLogProxy):
+
+    objects = TimelineLogProxyManager()
+
+    class Meta:
+        proxy = True
+        verbose_name = _("avg timeline log entry")
+        verbose_name_plural = _("avg timeline log entries")
```

Comment:
This is for the point `These logs need a special view in the admin to see them (filtered on above label)`.

To me it seemed that the best way to accomplish this was to create another proxy model which can then be used in the admin.  Maybe there is a better way?